eks-anywhere-common/Addons/Partner/Sysdig/external-secret.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: sysdig-secretstore-ekssnow
+  namespace: sysdig-agent
```

Comment:
could we keep namespace as `sysdig` throughtout for consistency?

Reply:
NS updated